docs/computed-fields.md

```diff
@@ -0,0 +1,65 @@
+# Computed Fields
+
+A card’s fields can link to or contain other cards. A special
```

Comment:
Is there a sentence fragment missing here?

Reply:
oops, I meant to replace that with the line below, removed now, thanks.